plugins/statistics.koplugin/main.lua

```diff
@@ -2973,6 +2974,23 @@ function ReaderStatistics:getCurrentBookReadPages()
     return read_pages
 end
 
+function ReaderStatistics:canSync()
+    return self.settings.sync_server ~= nil and self.settings.is_enabled
+end
+
+function ReaderStatistics:onSyncBookStats()
+    if not self:canSync() then return end
+
+    UIManager:show(InfoMessage:new {
+        text = _("Syncing book statistics, this might take a while…"),
```

Comment:
The `.` vs `,` is for obvious reasons, the `might` vs `may` because it's been phrased that way a few times before within the codebase.
```suggestion
        text = _("Syncing book statistics. This may take a while."),
```

Reply:
Applied. I've taken the `...` ending from highlight exporter plugin's in progress message ;)